src/azurelinux/3.0/net10.0/cross/loongarch64/Dockerfile

```diff
@@ -29,7 +32,7 @@ RUN TARGET_TRIPLE="loongarch64-linux-gnu" && \
         -DLLVM_ENABLE_RUNTIMES="libcxx;compiler-rt" \
         -DLIBCXX_ENABLE_SHARED=OFF \
         -DLIBCXX_CXX_ABI=libstdc++ \
-        -DLIBCXX_CXX_ABI_INCLUDE_PATHS="$ROOTFS_DIR/usr/include/c++/14/;$ROOTFS_DIR/usr/include/$TARGET_TRIPLE;$ROOTFS_DIR/usr/include/$TARGET_TRIPLE/c++/14" \
```

Comment:
just to make sure: we don't need the additional `$ROOTFS_DIR/usr/include/$TARGET_TRIPLE/c++/14` ?

Reply:
That's correct. It is on the same plan as riscv64 (both use python script to unpack deb files). The criteria is green CI here; this layer is invisible to end user, as it produces libcxx (with these include paths) which is copied to next layer that is actually exported.